backend/tests/v2/users/test_antisybil_status.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime, timedelta, timezone
+from http import HTTPStatus
+from fastapi import FastAPI
+import pytest
+from httpx import AsyncClient
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.constants import (
+    LOW_UQ_SCORE,
+    MAX_UQ_SCORE,
+    NULLIFIED_UQ_SCORE,
+    UQ_THRESHOLD_MAINNET,
+)
+from v2.uniqueness_quotients.dependencies import get_uq_score_getter
+from v2.uniqueness_quotients.services import UQScoreGetter
+from tests.v2.factories import FactoriesAggregator
+
+
+@pytest.mark.asyncio
+async def test_get_antisybil_status_for_user_happy_path(
```

Comment:
wondering if we shouldn't create separate tests for different cases, not put everything in a single UT, e.g `test_get_antisybil_status_when_user_on_a_timeout_list`, `test_get_antisybil_status_when_user_on_a_guest_list` etc.